app/controllers/rooms_controller.rb

```diff
@@ -4,6 +4,7 @@ def new
 
   def create
     room = Room.new(room_params)
+    room.choices.build(title: params[:room][:choice])
```

Comment:
Well what even is a room with one choice? One choice is no longer a choice. If we are going to assert that rooms _must_ be created with choices, then we should assert it must have at least _two_ choices. Otherwise we should remove the validation completely and just let them create rooms and add choices as needed.